dockerfiles/base/scripts/base/startup_01_init.sh

```diff
@@ -158,12 +158,8 @@ init_global_vars() {
   DEFAULT_CHE_LICENSE=false
   CHE_LICENSE=${CHE_LICENSE:-${DEFAULT_CHE_LICENSE}}
 
-  if [[ "${CHE_CONTAINER_NAME}" = "${CHE_MINI_PRODUCT_NAME}" ]]; then   
-    if [[ "${CHE_PORT}" != "${DEFAULT_CHE_PORT}" ]]; then
+  if [[ "${CHE_CONTAINER_NAME}" = "${CHE_MINI_PRODUCT_NAME}" ]]; then
       CHE_CONTAINER_NAME="${CHE_CONTAINER_PREFIX}-${CHE_PORT}"
-    else 
-      CHE_CONTAINER_NAME="${CHE_CONTAINER_PREFIX}"
```

Comment:
AFAIK this is changing the behavior

From what I'm seing with these changes che server will be named `che-8080` by default instead of only `che` as before

Reply:
that's right! 
I was waiting for this question :)
I just wanted to have less if statements and more clear flow.
So how do you think is it that critical for us to keep 'default' `che` name if port is default?
The issue isthat I don't have access in erb template to `$DEFAULT_CHE_PORT` so I will use 8080 to check which is not that good...

Reply:
yes, it's horrible to see `che-8080` for end user instead of `che`
(and if there are some user custom scripts, it will make them fail as it will expect to find a `che`container)

I see the issue but current fix is very odd to me.
AFAIK you can have all properties that you want inside erb templates.
BTW, can't we set the network name with docker compose file, so we can keep `che`container name but network will have the port prefix

Reply:
Ok so I will handle default values 
compose will always prepend network name with compose project name to avoid conflicts, and if specify network name in compose file compose will not try to create  that network it will expect that it is already exist.